Configuration/src/main/java/io/deephaven/configuration/Configuration.java

```diff
@@ -175,6 +277,10 @@ public String reloadProperties() throws IOException, ConfigurationException {
         return reloadProperties(false);
     }
 
+    String determinePropertyFile() {
```

Comment:
Why not make this abstract and have it be overridden by `DefaultConfiguration`?

Reply:
I was worried about making Configuration itself abstract and breaking other stuff.  Probably overly cautions. I'll just do this